src/ILCompiler.RyuJit/src/JitInterface/CorInfoImpl.RyuJit.cs

```diff
@@ -5,20 +5,482 @@
 using System;
 using System.Diagnostics;
 
+using Internal.IL;
 using Internal.TypeSystem;
 
 using ILCompiler;
+using ILCompiler.DependencyAnalysis;
 
 namespace Internal.JitInterface
 {
-    partial class CorInfoImpl
+    unsafe partial class CorInfoImpl
     {
+        private const CORINFO_RUNTIME_ABI TargetABI = CORINFO_RUNTIME_ABI.CORINFO_CORERT_ABI;
+
         private Compilation _compilation;
 
         public CorInfoImpl(Compilation compilation, JitConfigProvider jitConfig)
             : this(jitConfig)
         {
             _compilation = compilation;
         }
+
+        private void ComputeLookup(ref CORINFO_RESOLVED_TOKEN pResolvedToken, object entity, ReadyToRunHelperId helperId, ref CORINFO_LOOKUP lookup)
```

Comment:
These implementations look the same - are you folding in their differences in a follow up commit?

Reply:
Yes, currently I'm just making a mechanical copy to make the git history cleaner. The actual reimplementation of the ReadyToRun variants will require first merging in my other pending PR regarding the preparatory changes and basically lump in all the rest because the typical RTR changes in CorInfoImpl regard redirecting the various calls to NodeFactory to use the special R2R helpers.